production.Dockerfile

```diff
@@ -23,8 +23,10 @@ COPY --chown=appuser:appuser --from=build /upaya .
 # Copy in whole source (minus items matched in .dockerignore)
 COPY --chown=appuser:appuser . .
 COPY --chown=appuser:appuser --from=build /usr/local/bundle/config /usr/local/bundle
-RUN mkdir -p /upaya/log ; chown appuser /upaya/log
+RUN mkdir -p /upaya/log /usr/local/share/aws ; chown appuser /upaya/log
 
+# update CA certs so that we can trust RDS
+RUN curl https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem > /usr/local/share/aws/rds-combined-ca-bundle.pem
```

Comment:
is there any sort of sanity check we can run here to make sure that it's a real `.pem` and not an HTML 404 or something?

Reply:
Good call.  I was assuming that the curl failing would bomb the build out, but let me test that assumption.

Reply:
My assumption was incorrect, so I put a check in.  Thanks for pointing that out!

Reply:
Thanks!